multi-node/aws/pkg/cluster/config_test.go

```diff
@@ -0,0 +1,188 @@
+package cluster
+
+import (
+	"bytes"
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+const (
+	testVer         = "1"
+	externalDNSName = "test-external-dns-name"
+	keyName         = "test-key-name"
+	region          = "test-region"
+	clusterName     = "test-cluster-name"
+)
+const minimalYamlTemplate = `externalDNSName: %s
+keyName: %s
+region: %s
+clusterName: %s
+`
+
+var minimalConfigYaml string = fmt.Sprintf(
+	minimalYamlTemplate,
+	externalDNSName,
+	keyName,
+	region,
+	clusterName,
+)
+
+func correctMinimalConfig() *Config {
```

Comment:
Given that we probably don't need the other functions that make use of this, we can simplify the minimalConfigYaml

I'd probably suggest getting rid of this function (no longer called). Then also the individual config consts + minimalConfigYaml + minimalConfigTemplate can just be converted into a simple 4 line string var with fixed test values.